packages/model-viewer/src/test/features/environment-spec.ts

```diff
@@ -83,10 +75,10 @@ suite('ModelViewerElementBase with EnvironmentMixin', () => {
     element.src = MODEL_URL;
     document.body.insertBefore(element, document.body.firstChild);
     await rafPasses();
-    expect(environmentChangeCount).to.be.equal(0);
+    expect(environmentChangeCount).to.be.eq(0);
     element.style.display = 'block';
     await waitForEvent(element, 'environment-change');
-    expect(environmentChangeCount).to.be.equal(1);
```

Comment:
Is there a difference between these, other than the amount of typing?

Reply:
Good question; I don't think so? I just changed it to be consistent with the style elsewhere.